src/main/java/frc/robot/classes/Auton.java

```diff
@@ -261,12 +261,12 @@ private Command getAutonSequence(){
             case ONESCORECHARGE6:
                 return
                     new SequentialCommandGroup(
-                        resetOdometry("1ScoreCharge2-6"),
+                        resetOdometry("1ScoreCharge-6"),
                         highTransitionSequenceCone(),
                         new StartEndCommand(() -> swerveSubsystem.setModuleStates(swerveSubsystem.convertToModuleStates(0, -0.1, 0)), () -> swerveSubsystem.stopModules()).withTimeout(0.5),
-                        resetOdometry("1ScoreCharge2-6"),
+                        resetOdometry("1ScoreCharge-6"),
                         highScoreSequenceCone(),
-                        swerveSubsystem.followTrajectory("1ScoreCharge2-6", getTrajectory("1ScoreCharge2-6"))
+                        swerveSubsystem.followTrajectory("1ScoreCharge-6", getTrajectory("1ScoreCharge-6"))
                         .deadlineWith(armSubsystem.goToStateDelay(ArmState.STOWED).andThen(new RepeatCommand(new InstantCommand(() -> armSubsystem.updateMotors())))),
```

Comment:
updateMotors should be moved here, if we are moving to periodic with updateMotors.